backend/conduit/articles/views.py

```diff
@@ -46,6 +46,24 @@ def get_articles(isPublished=None, tag=None, author=None, favorited=None, limit=
     return res.offset(offset).limit(limit).all()
 
 
+#Route to return articles authored by current user
+@blueprint.route('/api/profile/articles', methods=('GET',))
```

Comment:
Move this route to the profile views.py file

Reply:
Done.